playground/backend/internal/code_processing/code_processing.go

```diff
@@ -109,9 +112,10 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 	runCmd := executor.Run(ctxWithTimeout)
 	var runError bytes.Buffer
 	runOutput := streaming.RunOutputWriter{Ctx: ctxWithTimeout, CacheService: cacheService, PipelineId: pipelineId}
+	go readLogFile(ctxWithTimeout, cacheService, lc, pipelineId, stopReadLogsChannel, finishReadLogsChannel)
 	runCmdWithOutput(runCmd, &runOutput, &runError, successChannel, errorChannel)
 
-	processStep(ctxWithTimeout, pipelineId, cacheService, cancelChannel, successChannel, nil, &runError, errorChannel, pb.Status_STATUS_RUN_ERROR, pb.Status_STATUS_FINISHED)
+	_ = processStep(ctxWithTimeout, pipelineId, cacheService, cancelChannel, successChannel, stopReadLogsChannel, finishReadLogsChannel, nil, &runError, errorChannel, pb.Status_STATUS_RUN_ERROR, pb.Status_STATUS_FINISHED)
```

Comment:
why do you ignore error here?

Reply:
In the previous implementation, this line was the last line of the method. So I just tried to avoid something like this:
```
if err := processStep(...); err != nil {
  return
}
```